install.sh

```diff
@@ -380,6 +386,9 @@ nvm_do_install() {
   echo "=> Close and reopen your terminal to start using nvm or run the following to use it now:"
   command printf "${SOURCE_STR}"
   if ${BASH_OR_ZSH} ; then
+    if [ -n "$ZSH_VERSION" ]; then
+        command printf "${ZSH_COMPLETION_STR}"
```

Comment:
2-space indentation

Reply:
Removed in favor of using `bash_completion`